<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MISHA Image Capture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="operationsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Operations</string>
    </property>
    <widget class="QPushButton" name="FocusButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Focus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CaptureButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture Raws</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FlatsButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture Flats</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TestLedsButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="text">
      <string>Test LEDs</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LightLevelsButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Light Levels</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CancelButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="LargeDisplay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>631</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="middleRightDisplay">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TopRightLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="TopRightDisplay">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="infobox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>380</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#infobox{
	background-color:white;
	border: 1px solid;
	border-color:#eeeeee;
}</string>
    </property>
    <property name="text">
     <string>Infobox Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="LightDisplayBR">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>311</width>
      <height>261</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="LightDisplayBL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>311</width>
      <height>261</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="LightDisplayTL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>311</width>
      <height>261</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="LightDisplayTR">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>311</width>
      <height>261</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as...</string>
     </property>
     <addaction name="actionCube"/>
    </widget>
    <addaction name="menuSave_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
